index.js

```diff
@@ -360,6 +360,32 @@ exports.ecsign = function (msgHash, privateKey) {
 }
 
 /**
+ * Returns the keccak-256 hash of `message`, prefixed with the header used by the `eth_sign` RPC call.
+ * The output of this function can be fed into `ecsign` to produce the same signature as the `eth_sign`
+ * call for a given `message`, or fed to `ecrecover` along with a signature to recover the public key
+ * used to produce the signature.
+ * @param message
+ * @returns {Buffer} hash
+ */
+exports.hashForEthSign = function (message) {
+  var prefix = exports.toBuffer('\u0019Ethereum Signed Message:\n' + message.length.toString())
+  return exports.sha3(Buffer.concat([prefix, message]))
+}
+
+/**
+ * ECDSA sign `message` using the same algorithm as the `eth_sign` RPC method.
+ * Prefixes message with fixed header + `message.length.toString()`, then
+ * signs the Keccak256 hash of the result.
+ * @param {Buffer} message
+ * @param {Buffer} privateKey
+ * @return {Object} signature
+ */
+exports.ethSign = function (message, privateKey) {
```

Comment:
I think naming this `ethSign` will cause a lot of confusion, given there's `ecsign` and this is not for signing transactions. Since it is such a slim helper, maybe it is not necessary to have it here?